dato.config.js

```diff
@@ -30,7 +30,18 @@ module.exports = (dato, root, i18n) => {
 }
 
 function appSettingsToJson(app) {
-  return pick(app, ['title', 'googleAnalyticsTrackingId'])
+  let appSettings = pick(app, ['title', 'googleAnalyticsTrackingId', 'themeColor', 'backgroundColor']);
+  appSettings = {
+    'title': appSettings.title,
+    'googleAnalyticsTrackingId': appSettings.googleAnalyticsTrackingId,
+    'themeColor': getRgbValue(appSettings.themeColor),
```

Comment:
Should colors be configurable from the CMS? There's no way to keep them in sync with other colors inside the project.

I see the icon is also configurable in the CMS, but not used here.